### ISSUE TITLE
local function clearTask() should be global

### ISSUE THREAD

**Issue by akbooer** [mysensors/Vera#15]
In a task error condition, this statement is executed:
luup.call_delay("clearTask", 15, "", false)

...but fails since the "clearTask" function scope is local, not global.

Solution: remove the 'local' in the function declaration!

**Comment by henrikekblad** [comment]
Ok! 

Guess you'll find more stuff when testing through your simulator. Will you create a PR against master when you're done?

**Comment by akbooer** [comment]
Yes - happy to do that.  There are a couple of other small things, but I want to do some real testing and that will have to wait (not too long) for me to build at least one real sensor.

**Comment by henrikekblad** [comment]
Hehe.. yeah, let me know if you need help with the sensor-building.. I will try to keep updated your forum thread on the vera-forum.
